README.md

```diff
@@ -45,8 +44,7 @@ tomorrowAsString(new Date('2015-01-01')); // '2015-01-02';
 
 ## API
 
-### parse
-`String -> Date`
+### parse :: String -> Date
```

Comment:
that is a valid point. im looking at putting the docs in a gitbook wich would change all of this